image.yaml

```diff
@@ -3,41 +3,56 @@ schema_version: 1
 name: "jboss-amq-6/amq63"
 description: "Red Hat JBoss AMQ 6.3 container image"
 version: "6.3.0"
-from: "jboss/openjdk18-rhel7:1.1-21"
+from: "rhel7:7-released"
 labels:
     - name: "com.redhat.component"
       value: "jboss-amq-6-amq63-container"
     - name: "org.jboss.product"
       value: "amq"
     - name: "org.jboss.product.version"
-      value: "6.3.0.redhat-347"
+      value: "6.3.0.redhat-356"
     - name: "org.jboss.product.amq.version"
-      value: "6.3.0.redhat-347"
+      value: "6.3.0.redhat-356"
 envs:
     - name: "JBOSS_PRODUCT"
       value: "AMQ"
     - name: "JBOSS_AMQ_VERSION"
-      value: "6.3.0.redhat-347"
+      value: "6.3.0.redhat-356"
     - name: "PRODUCT_VERSION"
-      value: "6.3.0.redhat-347"
+      value: "6.3.0.redhat-356"
     - name: "AMQ_HOME"
       value: "/opt/amq"
 modules:
       repositories:
           - path: modules
+          - name: cct_module
+            git:
+              url: https://github.com/jboss-openshift/cct_module.git
+              ref: master
       install:
+          - name: jboss.container.openjdk.jdk
+            version: "8"
           - name: amq-install
           - name: amq-chown
 artifacts:
-    - path: jboss-a-mq-6.3.0.redhat-347.zip
-      description: "Artifact is available on Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=59151&product=jboss.amq.broker&version=6.3.0&downloadType=securityPatches"
-      md5: d26ed42769c93d90ad8fbe9d0a217105
+    - path: jboss-a-mq-6.3.0.redhat-356.zip
```

Comment:
Optional: use `name` instead of `path`. This way the artifact is fetched directly from Brew and there is no need from cacher. More on the blog.